projects/ion/src/lib/dropdown/dropdown.component.ts

```diff
@@ -27,9 +27,11 @@ export class IonDropdownComponent implements OnInit, AfterViewInit {
   clearButtonIsVisible: boolean;
 
   setClearButtonIsVisible(): void {
-    this.clearButtonIsVisible = this.options.some(
-      (option) => option.selected === true
-    );
+    if (this.multiple) {
+      this.clearButtonIsVisible = this.options.some(
+        (option) => option.selected === true
```

Comment:
```suggestion
        (option) => option.selected
```
This can be removed, since selected is already a boolean.